src/harness/sas_testcase.py

```diff
@@ -45,3 +53,50 @@ def assertValidResponseFormatForApprovedGrant(self, grant_response):
     # operationParam should not be set if grant is approved
     self.assertFalse('operationParam' in grant_response)
     self.assertTrue(grant_response['channelType'] in ('PAL', 'GAA'))
+
+  def assertRegistered(self, registration_request,
+                       conditional_registration_data=None):
+    for device in registration_request:
+      self._sas_admin.InjectFccId({'fccId': device['fccId']})
+    if conditional_registration_data:
+      self._sas_admin.PreloadRegistrationData(conditional_registration_data)
+
+    request = {'registrationRequest': registration_request}
+    response = self._sas.Registration(request)['registrationResponse']
+
+    # Check the registration response; collect CBSD IDs
+    cbsd_ids = []
+    for resp in response:
+      self.assertEqual(resp['response']['responseCode'], 0)
```

Comment:
Do we also need a check for measReportConfig ? We can add a check based on the Registration Request Dictionary; if the measCapability is empty then the measReportConfig should not exist in Response?